http/polytracker/config.json5

```diff
@@ -0,0 +1,91 @@
+          "RUN mv example_picohttpparser.instrumented example_picohttpparser_track",
+          "WORKDIR /polytracker/examples/http/httpd",
+          "RUN git clone --branch 2.4.13 --single-branch https://github.com/apache/httpd.git",
+          "WORKDIR /polytracker/examples/http/httpd/httpd",
+          "RUN mkdir -p srclib/apr srclib/apr-util srclib/pcre srclib/expat",
+          "RUN curl https://archive.apache.org/dist/apr/apr-1.7.0.tar.gz -o apr-1.7.0.tar.gz",
+          "RUN tar xfz apr-1.7.0.tar.gz -C srclib/apr --strip-components 1 && rm apr-1.7.0.tar.gz",
+          "RUN curl https://archive.apache.org/dist/apr/apr-util-1.6.1.tar.gz -o apr-util-1.6.1.tar.gz",
+          "RUN tar xfz apr-util-1.6.1.tar.gz -C srclib/apr-util --strip-components 1 && rm apr-util-1.6.1.tar.gz",
+          "RUN curl -L https://sourceforge.net/projects/pcre/files/pcre/8.39/pcre-8.39.tar.gz/download -o pcre-8.39.tar.gz",
+          "RUN tar xfz pcre-8.39.tar.gz -C srclib/pcre --strip-components 1 && rm pcre-8.39.tar.gz",
+          "RUN curl -L https://github.com/libexpat/libexpat/releases/download/R_2_4_7/expat-2.4.7.tar.gz -o expat-2.4.7.tar.gz",
+          "RUN tar xfz expat-2.4.7.tar.gz -C srclib/expat --strip-components 1 && rm expat-2.4.7.tar.gz",
+          "WORKDIR /polytracker/examples/http/httpd/httpd/srclib/pcre",
+          "RUN polytracker build ./configure --disable-shared",
+          "RUN polytracker build make",
+          "WORKDIR /polytracker/examples/http/httpd/httpd/srclib/expat",
+          "RUN polytracker build ./configure --disable-shared",
+          "RUN polytracker build make",
+          "WORKDIR /polytracker/examples/http/httpd/httpd",
+          "RUN polytracker build ./buildconf",
+          "RUN CFLAGS=\"-I$(pwd)/srclib/pcre -I$(pwd)/srclib/expat/lib\" LDFLAGS=\"-L$(pwd)/srclib/pcre/.libs -L$(pwd)/srclib/expat/lib/.libs\" polytracker build ./configure --disable-shared --with-mpm=prefork --with-pcre=srclib/pcre/pcre-config --with-included-apr --enable-mods-static='authz_core unixd'",
+          "RUN CFLAGS=\"-I$(pwd)/srclib/pcre -I$(pwd)/srclib/expat/lib\" LDFLAGS=\"-L$(pwd)/srclib/pcre/.libs -L$(pwd)/srclib/expat/lib/.libs\" polytracker build make -j$((`nproc`+1))",
+          "RUN polytracker instrument-targets --taint --ftrace httpd",
+          "RUN mv httpd.instrumented httpd_track",
+          "RUN cp httpd_track httpd",
+          "RUN polytracker build make install"
+        ],
+      },
+      final: {
+        commands: [
+          "RUN apt-get update && apt-get -y upgrade && apt-get install -y git pkg-config netcat",
+          "WORKDIR /",
+          "RUN git clone --branch 6548-post-processing-cluster-input-bytes --single-branch https://github.com/trailofbits/polytracker.git",
+          "WORKDIR /polytracker",
+          "RUN pip3 install graphtage",
+          "RUN pip3 install .",
+          "WORKDIR /",
+          "RUN mkdir -p /usr/local/apache2/htdocs",
+          "RUN mkdir -p /usr/local/apache2/logs",
```

Comment:
Does polytracker's version of apache actually use the default apache config dirs and/or log dirs? I haven't seen it write to this log dir, and if it doesn't, that could simplify the changes I plan to make to my apache parser's configuration.

Reply:
It's not _intended_ to use the default config dirs. We specifically tried to have it be independent of any "real" apache instance. I _believe_ we had to add those `mkdir` lines to the `Dockerfile` because `apache` looks for their existence regardless of whether it uses them, but I am not 100% sure.

Reply:
Thanks for confirming. I wasn't able to find evidence of the default config bleeding in in my testing (I said before that I thought there was some shared config; I was wrong about that)